cours/containers/kubernetes/k8s-architecture.fr.md

```diff
@@ -0,0 +1,132 @@
+# Kubernetes : Architecture {-}
+
+### Kubernetes : Composants
+
+- Kubernetes est écrit en Go, compilé statiquement.
+
```

Comment:
I'd not put blank lines between different points in a single slide, just for readability of the md file